### ISSUE TITLE
elevated forced expiratory volume in one second

### ISSUE THREAD

**Issue by slaulederkind** [obophenotype/human-phenotype-ontology#6094]
**Preferred term label:**

Elevated forced expiratory volume in one second

**Synonyms**

Elevated FEV1

**Definition (free text, please give PubMed ID)**

An increase in the amount of air a person can forcefully expel in one second, compared to some standard or previous measure of same subject.  PMID:19671143

**Parent term (use hpo.jax.org/app)**

Abnormal spirometry test (HP:0032340)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

0000-0001-5361-6739

**Comment by pnrobinson** [comment]
sorry for the delay, added term.
